src/ansys/edb/core/definition/padstack_def_data.py

```diff
@@ -473,3 +503,49 @@ def solder_ball_material(self, material):
                 self, material
             )
         )
+
+    def get_connection_pt(self, layer):
+        """
+        Get connection point position and direction by layer name.
+
+        Parameters
+        ----------
+        layer : str
+            Layer name.
+
+        Returns
+        -------
+        tuple[:class:`Value <ansys.edb.core.utility.Value>`, \
+        :class:`Value <ansys.edb.core.utility.Value>`, :class:`ConnectionPtDirection`]
+
+        The tuple is in a ``(x, y, direction)`` format:
+
+        - ``x``: X of the connection point.
+        - ``y``: Y of the connection point.
+        - ``direction``: Direction of the connection point.
+        """
+        msg = self.__stub.GetConnectionPt(
+            _PadstackDefDataQueryBuilder.padstack_def_data_get_connection_pt_message(self, layer)
+        )
+        return Value(msg.x), Value(msg.y), ConnectionPtDirection(msg.direction)
```

Comment:
maybe return `PointData` instead of `Value`?

there are utility methods `conversions.to_point` or `parser.to_point_data` depending on your input